### PR TITLE
feat(cozy-logger): Add basic typing

### PR DESCRIPTION
**Pull request by acezard**
This will make using this module easier in TS envs

### CODE REVIEW

File: packages/cozy-logger/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  declare namespace log {
+    export const addFilter: (filter: unknown) => number
+
+    export const namespace: (
+      namespace: string
+    ) => (type: string, message: string, label?: string, ns?: string) => void
+
+    export const setNoRetry: (obj: unknown) => unknown
+
+    export const setLevel: (lvl: unknown) => void
```
Comment (trollepierre) [inline packages/cozy-logger/index.d.ts:18]: so sad to see a lot of unknown

**Review by trollepierre — APPROVED** [review]
I approved, because it's a very important first step, thanks @acezard !

File: packages/cozy-logger/index.d.ts (hunk 1 above)
Comment (acezard) [reply packages/cozy-logger/index.d.ts:18]: Correct but I'm not sure what the types should be unless I spend too much time on it

File: packages/cozy-logger/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
```
[the remark this answers is not in the document]
Comment (trollepierre) [reply packages/cozy-logger/tsconfig.json:3]: nit: yes

File: packages/cozy-logger/index.d.ts (hunk 1 above)
Comment (trollepierre) [reply packages/cozy-logger/index.d.ts:18]: it was a nit sorry

**Pull request merged**